### ISSUE TITLE
Auto-Completion not working

### ISSUE THREAD

**Issue by MicroPeenModding** [bridge-core/editor#1093]
**Summary**
1 week ago I was working on my project and I had auto completions for server, server-ui. Yesterday I loaded up my project and the auto completions wasn't working. I updated Bridge with no change, I uninstalled and reinstalled, started a fresh project and still no auto-completions. I have just now tried the Web App and have ran into the same issue as seen below.

**To Reproduce**

1. Create a project
2. Setup manifest and import "@minecraft/server"
3. Begin typing "server."
4. Observe result

**Observed behavior**
I was creating a script and noticed I had no auto-completion
**Expected behavior**
Auto-Completion prompts, what type the object is.

**Screenshots / File Attachments**
![image](https://github.com/bridge-core/editor/assets/77970897/3a1ad206-4716-4266-95d3-dbda3abfbd5b)
![image](https://github.com/bridge-core/editor/assets/77970897/bc857fd0-94be-4f2b-bf99-42c8c7882d8c)

**Platform (please complete the following information):**

-   OS: Windows 11 / Web App
-   App Version: 2.7.12 originally, also occurring on 2.7.14

**Additional context**

**Comment by outercloudstudio** [comment]
Thank you for reporting I'll take a look.

**Comment by Ch36** [comment]
Am having the same issue, I was going to create a new issue but looks like I dont have to. Please fix 🙏

**Comment by outercloudstudio** [comment]
Do you have any experiments enabled in the project config?

**Comment by Ch36** [comment]
> Do you have any experiments enabled in the project config?

Yes. Everything worked fine before 2.7.13 I think, I’m not entirely sure that is the correct version.
